gnomad/utils/filtering.py

```diff
@@ -383,6 +383,20 @@ def filter_to_clinvar_pathogenic(
     return t
 
 
+def filter_gencode_to_cds(ht: hl.Table) -> hl.Table:
```

Comment:
Can we wait to add this in a PR that uses it?

Reply:
I'm using it in the tx_annotate PR.

Reply:
because in v2, we filtered to CDS regions by default.

Reply:
that's fine, let's get this PR in and move this function to that PR then